scripts/mapping.js

```diff
@@ -419,7 +425,9 @@ var createCookie = function (name, value, days) {
           b = randInt(0, specialContents.length - 1);
           c = "<li>Special: " + specialContents[b] + "</li>";
         }
-        $("#roomcont").append(c);
+        var wrapper = document.createElement('div');
```

Comment:
I think this would make more sense if the wrapper element were the `<li>` itself and that tag were removed from the content string.